core/services/workflows/engine.go

```diff
@@ -853,7 +854,7 @@ func (e *Engine) Close() error {
 }
 
 type Config struct {
-	Spec                 string
```

Comment:
"Spec" feels OK, why the change?

Reply:
it wasn't used and spec is't always going to be a string.